spec/rubocop/cop/sorbet/sigils/enforce_sigil_order_spec.rb

```diff
@@ -143,5 +143,50 @@ class Foo; end
           class Foo; end
         RUBY
     end
+
+    it('autocorrects magic comments while removing blank lines') do
+      source = <<~RUBY
+        # frozen_string_literal: true
+
+
+        # typed: true
+
+
+
+        # encoding: utf-8
+        class Foo; end
+      RUBY
+      expect(autocorrect_source(source))
+        .to(eq(<<~RUBY))
+          # encoding: utf-8
+          # typed: true
+          # frozen_string_literal: true
```

Comment:
```suggestion
          # encoding: utf-8
          #
          # typed: true
          #
          # frozen_string_literal: true
```

Reply:
I can turn these 2 suggestions into a new test?

Reply:
Sure, that would be nice.